### DISCUSSION TITLE
What should I choose when I development environment?

### DISCUSSION THREAD

**Discussion by sgalcheung** [withastro/roadmap#1018] [Proposal]
# Background & Motivation

Are you confused about how I add an adapter to the development environment, this time we aren't sure how to finally deploy the platform, so how can we handle this situation?
Because the different render model has different handling codes, such as handling dynamic routes, when I confirm I'll enable the SSR model, I don't need to use `getStaticPaths()` method.

**Comment by sgalcheung** [comment]
I noticed the run time does not require an adapter, when run `npm run start` it will work well.

**Comment by ematipico** [comment]
Please use Discord for support and questions: https://astro.build/chat

**Reply by sgalcheung** [reply]
I move to Discord at https://discord.com/channels/830184174198718474/1284747674527993967
